testing/pacparser/APKBUILD

```diff
@@ -0,0 +1,34 @@
+# Contributor: Aaron Hurt <ahurt@ena.com>
+# Maintainer: Aaron Hurt <ahurt@ena.com>
+pkgname=pacparser
+pkgver=1.3.6
+pkgrel=0
+pkgdesc="A library to parse proxy auto-config (PAC) files"
+url="http://pacparser.manugarg.com"
+arch="all"
+license="LGPL3+"
+depends=""
+makedepends="bash"
+install=""
+subpackages="$pkgname-doc $pkgname-dev"
+source="$pkgname-$pkgver.tar.gz::https://github.com/pacparser/pacparser/archive/$pkgver.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver"
+
+prepare() {
+	cd "$builddir"
+	find . -name \*.git\* -delete
+}
+
+build() {
+	cd "$builddir"
+	make -C src pactester > /dev/null 2>&1 || return 1
+}
+
+package() {
+	cd "$builddir"
+	make -C src install DESTDIR=$pkgdir LIB_PREFIX=$pkgdir/usr/lib > /dev/null 2>&1 || return 1
```

Comment:
It's always better to have a lot of output than none. How would you know what's wrong if the package fail to build, when you throw away even error messages?